cfgov/v1/models/molecules.py

```diff
@@ -53,6 +55,8 @@ class TextIntroduction(blocks.StructBlock):
     links = blocks.ListBlock(atoms.Hyperlink(required=False), required=False)
     has_rule = blocks.BooleanBlock(required=False)
 
+    js = "expandable.js"
```

Comment:
for now it lives in Text Introduction as a proof of concept will later be removed when https://github.com/cfpb/cfgov-refresh/pull/1251 is merged